### PR TITLE
Twitter posts request example in main readme and travis, code climate ba...

### PR DESCRIPTION
**Pull request by kremez19**
...dges on top

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+[![Build Status](https://travis-ci.org/jnunemaker/httparty.svg?branch=master)](https://travis-ci.org/jnunemaker/httparty)
```
Comment (jnunemaker) [inline README.md]: I'm not a fan of these images really.

File: README.md [hunk 2]
```diff
@@ -45,7 +49,43 @@ puts stack_exchange.questions
 puts stack_exchange.users
 ```
 
-See the [examples directory](http://github.com/jnunemaker/httparty/tree/master/examples) for even more goodies.
+## Post request examples
```
Comment (jnunemaker) [inline README.md]: I'd rather just move this to an example than make the readme longer.

**Comment by kremez19** [comment]
Ok, thank you for you comments. I only found that it is useful to have some most common examples at the readme. Badges can be good indicators of project current state. Also I make
### See more examples section
to have more large font size.

You can close PR if you find this enhancements useless.

**Comment by jnunemaker** [comment]
I appreciate the intent, I just don't like the implementation. I'd accept a PR that just added the post example, maybe to basic.rb or a new file like post.rb.

**Pull request closed without merging**